docker/init/03-users.js

```diff
@@ -1,75 +1,25 @@
-const users = db.getSiblingDB('trello-clone')
-
 const user1 = {
   _id: ObjectId('6230ef5c91ecd051dc75df37'),
-  username: 'Marek Kowalski',
+  username: 'Marek Kowalski2137',
```

Comment:
Do we need to change this username?

Reply:
Hahah my bad that's a leftover from testing 😅